boilerplate/app/components/Toggle.tsx

```diff
@@ -184,13 +228,13 @@ function Checkbox(props: ToggleInputProps) {
   } = props
 
   const offBackgroundColor = [
-    disabled && colors.palette.neutral600,
+    disabled && colors.palette.neutral400,
     status === "error" && colors.errorBackground,
     colors.palette.neutral200,
   ].filter(Boolean)[0]
 
   const outerBorderColor = [
-    disabled && colors.border,
+    disabled && colors.palette.neutral400,
```

Comment:
@yulolimum just curious about this. We had a conversation about using palette vs semantic name on my design update PR. Would this not be a use case for `colors.border` since it's being used in the `outerBorderColor`?

Reply:
@cdanwards I saw that convo and I'm torn tbh. I made this change for the sake of consistency since we reference the palette for most other colors inside components. Honestly, I think either approach works _for now_. The reason I say _for now_ is because one of my next tasks will be to add an additional layer to styling to support multiple themes - a color mapping file. So, all of these color references will change in the coming weeks anyways.